lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryImpl.java

```diff
@@ -707,17 +707,17 @@ public TaskQuery orderByClaimed(SortDirection sortDirection) {
   @Override
   public TaskQuery orderByClassificationKey(SortDirection sortDirection) {
     return DB.DB2.dbProductId.equals(getDatabaseId())
-        ? addOrderCriteria("TCLASSIFICATION_KEY", sortDirection)
-        : addOrderCriteria("t.CLASSIFICATION_KEY", sortDirection);
+               ? addOrderCriteria("TCLASSIFICATION_KEY", sortDirection)
```

Comment:
Why do we have to change TaskQueryImpl in this PR?

Reply:
We don't. 
Thank you for pointing our. I'm reverting this.